spec/app2_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'chefspec'
+require 'chefspec/berkshelf'
+
+describe 'osl-app::app2' do
+  context 'on centos 7.2' do
+    let(:chef_run) do
+      ChefSpec::SoloRunner.new(platform: 'centos',
+                               version: '7.2.1511').converge(described_recipe)
+    end
```

Comment:
Take a look at what I did in ``app1_spec.rb`` and add it here so the test works.